node/pkg/lifecycle/startup/startup_test.go

```diff
@@ -248,6 +255,13 @@ var _ = Describe("FV tests against a real etcd", func() {
 						Expect(pool.Spec.IPIPMode).To(Equal(api.IPIPMode(expectIpv4IpipMode)))
 					}
 
+					// off is not a real mode value but use it instead of empty string
+					if expectIpv4VXLANMode == "Off" {
+						Expect(pool.Spec.VXLANMode).To(Equal(api.VXLANModeNever))
+					} else {
+						Expect(pool.Spec.VXLANMode).To(Equal(api.VXLANMode(expectIpv4VXLANMode)))
+					}
```

Comment:
Do we need `expectIpv4VXLANMode == "Off"`? I'm wondering if we can replace the if/else with:
```
Expect(pool.Spec.VXLANMode).To(Equal(api.VXLANMode(expectIpv4VXLANMode)))
```

Reply:
Same thing with `expectIpv6VXLANMode`

Reply:
I think we would still need some special logic because we consider "", "off", "Off" the same as "Never", "never" (I'm not sure if the existing logic is the best, I ended up copying it from the v4 IPIP code...